src/relay/chit_brickFamiliar.ash

```diff
@@ -967,7 +967,7 @@ void FamVampyre() {
 		result.append('</tbody></table>');
 	}
 
-	matcher id = create_matcher('https://s3.amazonaws.com/images.kingdomofloathing.com/(.+?)><br>(.+?)<br><font color=blue><b>(.+?)</font>', chitSource["familiar"]);
+	matcher id = create_matcher('https://d2uyhvukfffg5a.cloudfront.net/(.+?)><br>(.+?)<br><font color=blue><b>(.+?)</font>', chitSource["familiar"]);
```

Comment:
It strikes me that

```java
matcher id = create_matcher('<img src=(?:.+?)/(adventureimages/.+?)><br>(.+?)<br><font color=blue><b>(.+?)</font>', chitSource["familiar"]);
```

would probably be better

Reply:
sure